xml/System.Net.WebSockets/ClientWebSocket.xml

```diff
@@ -369,7 +369,17 @@
       <Docs>
         <param name="uri">The URI of the WebSocket server to connect to.</param>
         <param name="cancellationToken">A cancellation token used to propagate notification that the operation should be canceled.</param>
-        <summary>Connects to a WebSocket server as an asynchronous operation.</summary>
+        <summary>
+			Connects to a WebSocket server as an asynchronous operation.
+			The request has default headers:
+			VersionPolicy = RequestVersionExact
```

Comment:
VersionPolicy is not an HTTP header. It is an application-level setting for `ClientWebSocket`. But we might want to still mention the behavior with words, like "By default, HTTP/1.1 will be used. To change HTTP version used or enable HTTP version upgrade/downgrade, see (xref to HttpVersion option) and (xref to VersionPolicy option)"

Reply:
Added